### PR TITLE
App Update to Python3 - MivisionX Inference Analyzer

### PR DESCRIPTION
**Pull request by LakshmiKumar23**
1. Upgraded app to Python3 - command line
```
python3 mivisionx_inference_analyzer.py
```
2. Calls updated model compiler
3. Changed indentation to spaces as discussed 
4. Fixes part of #326

### CODE REVIEW

**Comment by LakshmiKumar23** [comment]
@kiritigowda Not sure how to fix the codacy issue. @rrawther said we can ignore it. Can you both have a look?

File: apps/mivisionx_inference_analyzer/mivisionx_inference_analyzer.py [hunk 1]
```diff
@@ -1,532 +1,534 @@
...
+        cv2.putText(resultImage,txt,(45,t_height+(topK*30+40)),cv2.FONT_HERSHEY_SIMPLEX,0.5,textColor,1)
+        topK = topK + 1
+    end = time.time()
+    if(verbosePrint):
+        print('%30s' % 'Processed results image in ', str((end - start)*1000), 'ms')
+
+    return resultImage, topIndex, topProb
+
+# MIVisionX Classifier
+if __name__ == '__main__':
```
Comment (LakshmiKumar23) [inline apps/mivisionx_inference_analyzer/mivisionx_inference_analyzer.py:146]: @kiritigowda Not sure how to fix the codacy issue. @rrawther said we can ignore it. Can you both have a look?

Comment (rrawther) [reply apps/mivisionx_inference_analyzer/mivisionx_inference_analyzer.py:146]: Not sure why it is complaining. But you could try putting all the code into a separate function and calling that to make it more simplified.

**Comment by kiritigowda** [comment]
@LakshmiKumar23 is this ready to be merged?

**Comment by LakshmiKumar23** [comment]
> @LakshmiKumar23 is this ready to be merged?

@kiritigowda Yes

**Review by kiritigowda — APPROVED** [review]

**Pull request merged**